config/application.rb

```diff
@@ -17,5 +17,6 @@ class Application < Rails::Application
     # the framework and any gems in your application.
 
     config.autoload_paths << Rails.root.join('app', 'poros')
+    config.time_zone = 'Buenos Aires'
```

Comment:
Guarda con esto, deberias de setear el timezone de la fecha y no de la aplicación entera

Reply:
Fixed. Removed this config line, and now we're setting up the time zone per request, in the rents controller, create method.

Reply:
fixed